src/Applications/LDAS_App/ldas_setup

```diff
@@ -462,6 +460,27 @@ class LDASsetup:
         tmp_expid = None
         tmp_expdir = None
 
+        my_ntasks_per_node = int(self.rqdRmInp['ntasks-per-node'])
+
+        # default number of nodes
+        my_nodes = self.rqdRmInp['ntasks_model'] / my_ntasks_per_node
+        if self.rqdRmInp['ntasks_model'] % my_ntasks_per_node > 0 :
+           my_nodes = my_nodes + 1
+
+        # over write above default if specified
```

Comment:
I don't understand the comment ("over write above ...") in relation to the subsequent if statement.
Should the comment read something like the following?
`# set oserver_nodes to 0 if not specified above`

Reply:
The comment is changed